src/worker/core/season/doAwards.ts

```diff
@@ -8,6 +9,10 @@ const doAwards = (conditions: Conditions) => {
 		return doAwardsFootball(conditions);
 	}
 
+	if (isSport("hockey")) {
```

Comment:
Change this to use `bySport` rather than `isSport`, which is what I've been doing to all the files like this when I add a .hockey.ts file. Reason is `bySport` will warn you if it's missing a sport, which would be handy if I add another in the future. It's pretty cool! `isSport` should only be used for things that would likely not need to be changed when adding another sport, like something very specific to only one sport.